.github/workflows/main.yml

```
@@ -22,7 +22,7 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Invoke downstream workflows
-        uses: keep-network/run-workflow@v1
+        uses: keep-network/ci/actions/run-workflow@move-actions-to-ci # TODO: change to @v1 before merging to main
```

Can we use the path to the current repository?
```suggestion
        uses: ./actions/run-workflow
```

---

As the `Main` workflow is used in our process of deployment on testnet, we want to use fixed version (tag) of the action here. Leaving as it is.